packages/driver/cypress/e2e/commands/assertions.cy.js

```diff
@@ -790,19 +789,18 @@ describe('src/cy/commands/assertions', () => {
       cy.noop({}).should('have.property', 'foo')
     })
 
-    it('ends and snapshots immediately and sets child', (done) => {
+    it('snapshots immediately and sets child', (done) => {
```

Comment:
This PR does introduce one test change - the assertion log is in a `pending` state when initially added, and is `passed` asynchronously as part of the `finishAssertions()` function, which happens slightly later in the lifecycle of the command.

Other tests verify that it moves from the `pending` state to `resolved` before the next command executes - it's just slightly later in resolving the same promise chain than `log:added`. I don't believe this is a meaningful difference, and so this can remain a chore even though I had to update one test. :woman_shrugging: